components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -133,6 +133,7 @@ class GeckoEngineSession(
      */
     override fun goBack() {
         geckoSession.goBack()
+        notifyObservers { onNavigateBack() }
```

Comment:
Can you add a small test for the `engine-gecko` components that verifies that calling `goBack()` will invoke the observer?

Reply:
Done :D